### PR TITLE
v0.8?

### PR DESCRIPTION
**Pull request by ghost**
I worked through some build issues when trying to build master off of windows 7 with build instructions given.  After the updates I ended up with a .nuget package that was usable and I was able to publish to our local repo.  I'm hoping these fixes can help improve the same type of impediments for you too so you might be able to publish to the main NuGet repo for everyone.

Primary reason I needed the updated build was a FluentNHibernate version conflict I had with Facility v0.7.1.  (RE: [SO](http://stackoverflow.com/questions/36454072/castle-facilities-nhibernate-and-fluentnhibernate-v2-x/36458813#36458813))

### CODE REVIEW

File: src/samples/NHibernate.AttrExampleConsoleApp/NHibernate.AttrExampleConsoleApp.csproj [hunk 1]
```diff
@@ -76,12 +88,17 @@
...
-    <Reference Include="System.Data.SQLite">
-      <SpecificVersion>False</SpecificVersion>
-      <HintPath>..\..\..\packages\System.Data.SQLite\lib\net40\System.Data.SQLite.dll</HintPath>
+    <Reference Include="System.Data.SQLite, Version=1.0.99.0, Culture=neutral, PublicKeyToken=db937bc2d44ff139, processorArchitecture=MSIL">
+      <HintPath>..\..\..\packages\System.Data.SQLite.Core.1.0.99.0\lib\net40\System.Data.SQLite.dll</HintPath>
+      <Private>True</Private>
     </Reference>
-    <Reference Include="System.Data.SQLite.Linq">
-      <HintPath>..\..\..\packages\System.Data.SQLite\lib\net40\System.Data.SQLite.Linq.dll</HintPath>
+    <Reference Include="System.Data.SQLite.EF6, Version=1.0.99.0, Culture=neutral, PublicKeyToken=db937bc2d44ff139, processorArchitecture=MSIL">
```
Comment (haf) [inline src/samples/NHibernate.AttrExampleConsoleApp/NHibernate.AttrExampleConsoleApp.csproj]: Why pull in EF?

**Comment by johnEsacks** [comment]
There were missing SQLite references preventing a compile.  When I pulled SQLite from NuGet that came with it, so I just left it in there.  No other reason.

File: src/samples/NHibernate.AttrExampleConsoleApp/NHibernate.AttrExampleConsoleApp.csproj (hunk 1 above)
Comment (johnEsacks) [inline src/samples/NHibernate.AttrExampleConsoleApp/NHibernate.AttrExampleConsoleApp.csproj]: There were missing SQLite references preventing a compile.  When I pulled SQLite from NuGet that came with it, so I just left it in there.  No other reason.

Comment (haf) [inline src/samples/NHibernate.AttrExampleConsoleApp/NHibernate.AttrExampleConsoleApp.csproj]: You can make it work with some of the parent packages of this one. You don't need to pull in EF...

**Pull request merged**